src/CmsManager/DecoratorStrategy.php

```diff
@@ -100,7 +100,7 @@ public function isRouteNameDecorable($routeName)
         }
 
         foreach ($this->ignoreRoutePatterns as $routePattern) {
-            if (preg_match(sprintf('#%s#', $routePattern), $routeName)) {
+            if (0 !== preg_match(sprintf('#%s#', $routePattern), $routeName)) {
```

Comment:
```suggestion
            if (1 === preg_match(sprintf('#%s#', $routePattern), $routeName)) {
```
false means an error